.env

```diff
@@ -1,7 +1,7 @@
 NEXT_PUBLIC_API_URL="https://api.wiiqare.com"
 CALLBACK_URL="https://app.wiiqare.com"
 
-NEXTAUTH_URL=https://app.wiiqare.com
+NEXTAUTH_URL=http://localhost:3000
```

Comment:
did you forgot to change this back?

Reply:
Sorry I forgot, did you change it or I do it ?

Reply:
you should change it